### PR TITLE
Properly handle error states when running schedule records interactively

### PR DESCRIPTION
**Pull request by borkweb**
See: https://central.tri.be/issues/66891

### CODE REVIEW

File: src/Tribe/Aggregator/Record/Abstract.php [hunk 1]
```diff
@@ -329,6 +329,15 @@ public function create_schedule_record() {
 		);
 
 		foreach ( $this->meta as $key => $value ) {
+			// don't propagate these meta keys to the scheduled record
+			if (
+				'preview' === $key
```
Comment (bordoni) [inline src/Tribe/Aggregator/Record/Abstract.php]: `in_array( $key, array( 'preview', 'activity', 'ids_to_import' ) )`??

File: src/Tribe/Aggregator/Tabs/Scheduled.php [hunk 2]
```diff
@@ -217,7 +217,10 @@ private function handle_get() {
 	 * @return string
 	 */
 	private function action_notice( $action, $ids = array(), $error = null ) {
-		$ids    = explode( ',', $ids );
+		if ( $ids && ! is_array( $ids ) ) {
+			$ids    = explode( ',', $ids );
```
Comment (bordoni) [inline src/Tribe/Aggregator/Tabs/Scheduled.php]: Odd spacing.

File: src/Tribe/Aggregator/Record/Abstract.php (hunk 1 above)
Comment (borkweb) [inline src/Tribe/Aggregator/Record/Abstract.php]: It is more efficient as `||` but I can switch it if it annoys you.

`¯\_(ツ)_/¯`

**Comment by bordoni** [comment]
👍

**Pull request merged**